docs/wiki/Managing-your-DCRDEX-Accounts.md

```diff
@@ -0,0 +1,126 @@
+This is set to zero when **Auto Renew** is disabled.
+* **Actual Tier:** Reflects your account's current actual trading tier with the DCRDEX server,
+determined based on your active [bonds](Creating-a-DCRDEX-Server-Account#fidelity-bonds) 
+and [reputation](#reputation-score) score.
+* **Penalties:** the number of tiers that are currently revoked due to low user score.
+* **Bonds Pending Refund:** expired bonds that are pending a refund transaction.
+* **Auto Reneww:** enable automatic bond renewal for the selected target tier.
+* **Penalty Comps:** enable posting additional bonds to offset penalized tiers.
+* **Reputation:** current [reputation](#reputation-score) score for the associated DCRDEX server account.
+
+# Changing Trading Tier
+
+Increasing your tier enables trading of larger quantities at a time. A higher tier also 
+increase your capacity for violations before trading privileges are completely suspended.
+Trading limits are also increased as you establish [reputation](#reputation-score) by engaging 
+in normal trading activity and successfully completing matches.
+
+<img src="./images/using-bison-wallet/settings-changeTier.png" width="400" alt="">
+
+Increasing your tier will require locking additional funds in a
+[fidelity bond](Creating-a-DCRDEX-Server-Account#fidelity-bonds), while lowering it 
+will reduce the amount required to renew your tier once the current bonds expire.
+
+# Reputation Score
+
+Users are incentivized to respond with their preimage for their submitted orders and to complete 
+swaps as the match negotiation protocol specifies. If they fail to act as required, their account
+will be penalized, with trading privileges eventually being completely suspended after multiple
+[revoked matches](Order-Management#revoked-matches).
+
+A suspended account will require time-locking additional funds on a new
+[fidelity bond](Creating-a-DCRDEX-Server-Account#fidelity-bonds) to offset penalized tiers.
+The DCRDEX scoring system has been designed to balance the need to deter intentional disruptions with the 
+reality of unreliable consumer networks and other such technical issues.
+
+Your reputation score increases by successfully completing trades.
+Misbehaviour and failure to act on a trade will penalize your score depending on the type of match failure:
+
+| Action             | Score Penalty | Notes                                                 |
+|--------------------|---------------|-------------------------------------------------------|
+| No Redeem as Taker | - 1           | Counterparty not affected, no stuck funds.            |
+| No Swap as Maker   | - 4           | Book spoof, taker needs a new order. No locked funds. |
+| Excessive Cancels  | - 5           |                                                       |
+| No Redeem as Maker | - 7           | Taker has funds stuck for 8 hours in contract.        |
+| No Swap as Taker   | - 11          | Maker funds stuck for 20 hours in contract.           |
```

Comment:
I think Taker Swap and Redeem are swapped.

https://github.com/decred/dcrdex/blob/504d3a9efbca333ad22e0b4af435c2c440f09131/server/auth/auth.go#L266-L283